api/stakeholder.go

```diff
@@ -188,16 +208,36 @@ func (r *Stakeholder) With(m *model.Stakeholder) {
 	r.Name = m.Name
 	r.Email = m.Email
 	r.JobFunction = r.refPtr(m.JobFunctionID, m.JobFunction)
+	r.Groups = []Ref{}
```

Comment:
hint: without this assignment, the nil list gets serialized to json as a `null`. Fixing this is intended to make the behavior consistent with the rest of the api.